src/main/index.ts

```diff
@@ -65,14 +65,23 @@ log.transports.file.format = "[{y}-{m}-{d} {h}:{i}:{s}.{ms}][{processType}][{lev
 log.eventLogger.format = "Electron event {eventSource}#{eventName} observed";
 
 log.hooks.push((message, transport) => {
-  if (transport !== log.transports.file) return message;
-
-  // Annoyingly spammed log from YTM that doesn't need to get added to the log file
-  if (message.data[0].includes("Third-party cookie will be blocked.")) return false;
-  for (let i = 0; i < message.data.length; i++) {
-    let dataString: string = message.data[i] as string;
-    dataString = dataString.replaceAll(/(?<=((https|http):\/\/)?.{1,64}(\..{1,64})?\..{1,64}\/)([\S]+)/gm, "[REDACTED]");
-    message.data[i] = dataString;
+  if (transport !== log.transports.file) { return message; }
```

Comment:
Not sure why you've re-added these brackets?
There's a few instances where I feel like brackets shouldn't be used, and this is a key one.

It's clear to see that there's only 1 thing running and it's escaping instantly.

Reply:
This is still not addressed